src/data/tests/ContentHighlightsTestData/utils.js

```diff
@@ -0,0 +1,12 @@
+import { camelCaseObject } from '@edx/frontend-platform';
+
+const rawTestCourseData = require('./testCourseData.json');
```

Comment:
Might recommend looking into `rosie` (https://github.com/rosiejs/rosie) to generate these test data via factories instead of hardcoded JSON files.